src/dev/src/app/datagrid/kitchen-sink/kitchen-sink.ts

```diff
@@ -74,7 +77,7 @@ export class DatagridKitchenSinkDemo {
     this.cleanUp();
   }
 
-  cleanUp(): void {
+  cleanUp() {
```

Comment:
Curious.. how does having the return type of `void` here impact/block AOT?

Reply:
It doesn't impact, but having it explicit is just extra verbosity that doesn't do anything for the compiler. So my guess is Cory might have used a slightly stricter linting config or something similar, that removed "useless" typings. I'm 👍 with that.

Reply:
It doesn't apply to AOT but I believe TypeScript will usually will infer void on a method without a return statement. I removed it as the rest of the methods did not use it. This was something I noticed going through the dev app, some methods use void while others did not. I was not sure what style the team prefers so I just made it consistent to the rest of the file.